docs/docs/guides/web3_config/index.md

```diff
@@ -201,6 +202,575 @@ console.log(web3.getContextObject().config)
 */
 ```
 
+### handleRevert
+When `handleRevert` is set to True, the following methods will retrieve specific error types and error messages:
```

Comment:
Can you mention within the configs what the default value is and setting that config within the code snippet

Reply:
Thanks for the review! Do you mean I should use the default value as the example within the code snippet? I mentioned the default value in the explanation for each config property. It may not be clear enough since some of them appear in the middle of a sentence. I can emphasize what the default value is in a single sentence above the code snippet. Is that better?